app/src/main/assets/applozic-settings.json

```diff
@@ -113,7 +113,8 @@
     ":location": true,
     ":camera": true,
     ":file": true,
-    ":audio": true
+    ":audio": true,
+    ":imageVideo": true
```

Comment:
What is image video combined? What if someone need to enable image but not video?

Reply:
the issue I was assigned included creating a new attachment system where you could select multiple images/videos and send them in one go.
setting each of these values to true enables the respective button. We have only one button for a combination of image/video.
Do you want me to change it in such a way that we can individually enable only image or video sending for that particular button? (i.e with image: true you can only select images, video: true only video and so on and if both are false the entire button will be hidden)

Reply:
@devashishmamgain what about this?

Reply:
It is related to allowing image and video or allowing multiple together?

Reply:
Multiple together. Setting this to true will make the button (that allows you to upload images and video in one go) visible.

Reply:
done